src/main/java/seedu/address/logic/commands/FilterCommand.java

```diff
@@ -20,29 +24,36 @@ public class FilterCommand extends Command {
             + "If there is no contact with the given tag, "
             + "an empty list of contacts will be displayed.\n"
             + "Ensure that tag only contains alphanumeric characters. \n"
-            + "Parameters: TAG\n"
+            + "Parameters: t/TAG (one or more)\n"
```

Comment:
for this, I think we can follow the same format as the AddCommand, where they use `...` to indicate one or more